### DISCUSSION TITLE
Usar alguma ferramenta para padronizar o código

### DISCUSSION THREAD

**Discussion by george-gca** [levxyca/diciotech#222] [Ideas]
Não sei qual ferramenta as pessoas usuárias do projeto costumam usar, num outro projeto que eu participo tenho usado o [prettier](https://prettier.io/). A maior vantagem pra mim dele é que é instalar e esquecer. Dá pra adicionar ele como uma ferramenta pre-commit, além de dar pra criar um GitHub action que valida que o código submetido está de acordo com os padrões dele.

**Comment by levxyca** [comment]
Legal @george-gca! Eu costumo usar o prettier também, gosto bastante! Acredito ser uma ótima solução pra gente tb!

**Reply by george-gca** [reply]
Beleza, criei a #240.
